pkg/pgmodel/migrations/sql/1_base_schema.up.sql

```diff
@@ -105,35 +108,34 @@ CREATE TRIGGER make_metric_table_trigger
 -- Internal functions --
 ------------------------
 
--- Return a table name built from a metric_name and a suffix.
--- The metric name is truncated so that the suffix could fit in full.
-CREATE OR REPLACE FUNCTION SCHEMA_CATALOG.metric_table_name_with_suffix(
-        metric_name text, suffix text)
+-- Return a table name built from a full_name and a suffix.
+-- The full name is truncated so that the suffix could fit in full.
+-- name size will always be 63 chars.
+CREATE OR REPLACE FUNCTION SCHEMA_CATALOG.pg_name_with_suffix(
```

Comment:
All of these naming things have a huge impact and we have to think about them ahead of time because if we screw it up the migration is a really big PITA.

Reply:
All of the name generation actually goes thru SCHEMA_CATALOG.pg_name_unique. The logic is that all names that have suffixes for uniqueness are alway 63 chars long, and a name that comes in without a suffix that is 63 chars long always has the suffix added, so as long as the suffix is unique there cannot be a conflict

Reply:
I think you're missing what I'm saying here. there's the possibility that someone uses the name that we came up with on their own.
so we end up generating a name that is then used by them. My thought is we can add a prefix so they can avoid that if needed.

Reply:
No there isn't that possibility because any name of length =63 will always have our suffix even if it originally had another suffix